server-plugins/activity-resources/src/utils.ts

```diff
@@ -296,6 +304,7 @@ export async function getTxAttributesUpdates (
     if (
       hierarchy.isDerived(attrClass, core.class.TypeMarkup) ||
       hierarchy.isDerived(attrClass, core.class.TypeCollaborativeMarkup) ||
+      hierarchy.isDerived(attrClass, core.class.TypeCollaborativeDoc) ||
```

Comment:
Looks like we dont need build documents diff by txes for `TypeCollaborativeDoc` because below you get it from storageAdapter

Reply:
oh, I see where it is used

Reply:
I need previous value to build diff. Can you suggest another way to do this?